lib/translator/common.h

```diff
@@ -23,6 +28,34 @@ void DebugLog(const char* format, ...);
 
 void SetDebugCallback(void (*callback)(const char*));
 
+void* AllocPage(uint8_t count);
+
+void SetAllocPageCallback(void* (*callback)(uint8_t));
+
+struct ScsiContext {
+  scsi_defs::LunAddress lun;
+};
+
+struct NvmeCompletionData {
+  nvme_defs::GenericQueueEntryCpl nvme_cpl[3];
+};
```

Comment:
Do we have strong reason to expect these will need multiple fields in the future? If not, the extra type seems like unnecessary boilerplate.